### ISSUE TITLE
string color hard to see color in dark theme

### ISSUE THREAD

**Issue by blaylockbk** [jupyterlab/jupyterlab#8158]
I love the dark theme, but the dark red string color is very difficult to see against the dark background (I have been accused of color blindness)

https://github.com/jupyterlab/jupyterlab/blob/083c65d92686d23b813f0242fca5be3d8b6fae37/packages/theme-dark-extension/style/variables.css#L352

Can it be changed to have a higher contrast ratio against the black background

from #ba2121 
![image](https://user-images.githubusercontent.com/6249613/78275733-d7238f80-74c6-11ea-9b0c-54f38e6a98ff.png)

to #ff7070
![image](https://user-images.githubusercontent.com/6249613/78275752-e0acf780-74c6-11ea-8914-209573a1969c.png)

---

I changed this in my personal environment, but thought it might be helpful for other people using Jupyter's Dark Theme.

Before...
![image](https://user-images.githubusercontent.com/6249613/78275943-2b2e7400-74c7-11ea-9795-85d141a28a88.png)

After...
![image](https://user-images.githubusercontent.com/6249613/78275968-397c9000-74c7-11ea-99d8-742af880f00e.png)

**Comment by jasongrout** [comment]
CC @tgeorgeux

**Comment by jasongrout** [comment]
See also https://github.com/jupyterlab/jupyterlab/pull/7779, which advocates for much larger changes. I agree the red is hard to read - perhaps just this could be changed with much less controversy?

**Comment by jasongrout** [comment]
> I changed this in my personal environment,

Just curious, how did you change this?

**Comment by blaylockbk** [comment]
I changed the line in the file

```
~/anaconda3/envs/best/share/jupyter/lab/themes/@jupyterlab/theme-dark-extension/index.css
```

**Comment by jasongrout** [comment]
Thanks. I was wondering if you had used an "official" way to change the color (like making your own theme), or if you just hacked it in.

**Comment by tgeorgeux** [comment]
That change looks great to me. I ran it through webAims contrast checker to be sure and it passes with flying colors!

![Screen Shot 2020-04-02 at 11 51 17 AM](https://user-images.githubusercontent.com/22109376/78290384-89178780-74d8-11ea-9524-2daa44e3b054.png)

**Comment by jasongrout** [comment]
@tgeorgeux - how about you make the final call on if this is the shade of red we want, and merge it if it's good, or if you want to tweak it more.

**Comment by tgeorgeux** [comment]
I think this color is better than what we're currently using, the changes in #7779 are larger and need more consideration, but for 2.1 this is a good change that we should implement. I can make a PR to try to get this in before Monday.

**Comment by blaylockbk** [comment]
Thanks for making the change! I think a lot of people will appreciate it.
